lib/plugins/aws/deploy/tests/mergeIamTemplates.js

```diff
@@ -121,4 +121,134 @@ describe('#mergeIamTemplates()', () => {
             .Resources.IamRoleLambdaExecution
         ).to.not.exist);
   });
+
+  it('should add a CloudWatch LogGroup resource', () => {
+    awsDeploy.serverless.service.provider.cf_logs = true;
+    const normalizedName = `${functionName[0].toUpperCase()}${functionName.substr(1)}LogGroup`;
+    return awsDeploy.mergeIamTemplates().then(() => {
+      expect(awsDeploy.serverless.service.provider.compiledCloudFormationTemplate
+        .Resources[normalizedName]
+      ).to.deep.equal(
+        {
+          Type: 'AWS::Logs::LogGroup',
+          Properties: {
+            LogGroupName: `/aws/lambda/${functionName}`,
+          },
+        }
+      );
+    });
+  });
+
+  it('should update IamPolicyLambdaExecution with a logging resource for the function', () => {
+    awsDeploy.serverless.service.provider.cf_logs = true;
+    const service = awsDeploy.serverless.service; // avoid 100 char lines below
```

Comment:
Could be rewritten like this so that it's consistent with our existing codebase:

```
awsDeploy.serverless.service
  .functions = { foo: bar };
```